src/main/java/com/nordstrom/kafka/connect/lambda/LambdaSinkConnectorConfig.java

```
@@ -26,6 +26,7 @@
   private static final int HTTP_PROXY_PORT_DEFAULT = -1;
   private static final boolean AWS_LAMBDA_BATCH_ENABLED_DEFAULT = true;
   private static final String AWS_LAMBDA_INVOCATION_MODE_DEFAULT = InvocationMode.SYNC.name();
+  private static final String AWS_LAMBDA_INVOCATION_FAILURE_DEFAULT = InvocationFailure.DROP.name();
```

Should the default be STOP? 

If someone doesn't know about the configuration, is it not better to default to a conservative scenario and adjust for efficiency? Just my opinion.

---

Hmm, that is a valid point.  I was initially of the view to just silently skip over failed message invocations and continue processing as the default (but yes, that kind of assumes the user is aware of this happening).  Someone not even knowing about the configuration and then wondering why some messages are not sent is entirely plausible.  I agree it would be better to be more conservative and just stop processing and allow the user to either determine if they want to drop and continue (via changing the setting) or if they want to reduce the message payload somehow.  I have changed the default to reflect this.